bpf/lib/common.h

```diff
@@ -71,12 +71,13 @@
 #define CILIUM_CALL_IPV6_NODEPORT_NAT		16
 #define CILIUM_CALL_IPV4_NODEPORT_REVNAT	17
 #define CILIUM_CALL_IPV6_NODEPORT_REVNAT	18
-#define CILIUM_CALL_ENCAP_NODEPORT_NAT		19
+#define CILIUM_CALL_IPV4_ENCAP_NODEPORT_NAT	19
 #define CILIUM_CALL_IPV4_NODEPORT_DSR		20
 #define CILIUM_CALL_IPV6_NODEPORT_DSR		21
 #define CILIUM_CALL_IPV4_FROM_HOST		22
 #define CILIUM_CALL_IPV6_FROM_HOST		23
-#define CILIUM_CALL_SIZE			24
+#define CILIUM_CALL_IPV6_ENCAP_NODEPORT_NAT	24
```

Comment:
Nit: I changed that while trying to debug the K8sUpdates test, but would be nice to move `CILIUM_CALL_IPV6_ENCAP_NODEPORT_NAT` back to be right after `CILIUM_CALL_IPV4_ENCAP_NODEPORT_NAT`.